### ISSUE TITLE
can't serve files with underscores

### ISSUE THREAD

**Issue by dvera** [dfletcher/tsws#3]
underscores in $URL_PATH get removed when tsws looks for the file?

**Comment by dfletcher** [comment]
Investigating this. While testing just noticed another bug with file handling, the TSWS_ROOT needs to be normalized before we start comparing it to the request path. I'll try to wrap up both of these problems today.

**Comment by dfletcher** [comment]
Fix is currently in branch `issue-3`, do a pull and then `git checkout issue-3`. It will be merged into master for v0.2 if there is no issues with it.

**Comment by dvera** [comment]
Working great for me. Thanks
